packages/create-cloudflare/src/frameworks/nuxt/index.ts

```diff
@@ -18,8 +19,10 @@ const generate = async (ctx: PagesGeneratorContext) => {
 	logRaw(""); // newline
 };
 
+
 const configure = async (ctx: PagesGeneratorContext) => {
 	process.chdir(ctx.project.path);
+	writeFile('./.node-version', '17');
```

Comment:
I thought of separating the node version thing so that if someone were to use C3 to create a nuxt app but link it to the Pages CI then they would get the node version set correctly (instead of either having to set it manually or having to add it to their build command)

Reply:
PS: 17 is also unsupported (https://endoflife.date/nodejs) so maybe we should just do 18? 🤷 

(also the build image v2 uses 18 by default I believe? so maybe there is no need to actually do anything? @GregBrimble)